### DISCUSSION TITLE
Learn keybinds

### DISCUSSION THREAD

**Discussion by noahfraiture** [zed-industries/zed#14936] [Feature Requests]
Hello, I have a problem
I aim to use zed editor with jetbrains base motion that I don't know yet very well and vim-mode (and soon helix mode when ready). But with all that there's no place to see the keybinds. [https://zed.dev/docs/key-bindings](officiel binds) this page isn't accurate anymore since it's only for vscode motion (or atom ?), this file [https://github.com/zed-industries/zed/blob/main/assets/keymaps/linux/jetbrains.json](jetbrains.json) and there's no documentation for the changes made by vim motion
Is there a way to learn the motion ? For instance jetbrains has an extension that make a notification when you use a button that you could have replaced by a keybind, it would be great to have something like that.
Do you have another solution ?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
